### ISSUE TITLE
watching a replay returns dropped weapons

### ISSUE THREAD

**Issue by Shegetz** [momentum-mod/game#1537]
### Describe the bug

When exiting a replay the weapons dropped by a player are returned to them

### How To Reproduce

1. join a bhop or surf map
2. drop any weapons
3. enter a replay
4. exit the replay
5. observe the weapons are back

### Expected Behavior

when exiting a replay a play that has dropped their weapons should not have them returned. Players do not want to have to repeatedly drop them

### Operating System

Windows 10

### Renderer

DX11

**Comment by Gocnak** [comment]
Yeah it should save weapon state upon entering replay. Could maybe modify the `SaveState_t` struct.

**Comment by Gocnak** [comment]
Does this still happen on latest 0.10?
